apps/web/src/design-system/icons/general/DigestGradient.tsx

```diff
@@ -1,19 +1,8 @@
 import React from 'react';
 
-export interface IIconProps {
-  width?: string;
-  height?: string;
-}
-/* eslint-disable */
-export function DigestGradient({ width = '30', height = '31' }: IIconProps) {
+export function DigestGradient(props: React.ComponentPropsWithoutRef<'svg'>) {
```

Comment:
allow to override the width and height like we do in other icons

Reply:
Wasn't working before?

Reply:
it was only allowing for the width/height, but not for ex. `styles`